Homework/Hoang_Nguyen/lesson3/215_findKthLargest.js

```diff
@@ -0,0 +1,37 @@
+function partition(nums, low, high) {
+  const pivot = nums[high];
```

Comment:
let's try use a random pivot?

Reply:
I updated both 2 solutions. Please review it.